src/Apps/Legion/Components/WizardShell/Internal/DataSourceStep/DataSourceStep.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import DataSourceStep from './DataSourceStep';
+import { IDataSourceStepProps } from './DataSourceStep.types';
+import { getDefaultStoryDecorator } from '../../../../../../Models/Services/StoryUtilities';
+
+const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
+
+export default {
+    title: 'Components/Apps/Legion/DataSourceStep',
```

Comment:
similar grouping here
```suggestion
    title: 'Components/Apps/Legion/WizardShell/Internal/DataSourceStep',
```